app/screens/home/recent_mentions/components/mention/mention.tsx

```diff
@@ -84,27 +97,46 @@ function Mention({post, currentUser}: Props) {
         />
     );
 
+    const handlePress = preventDoubleTap(() => {
+        pressDetected.current = true;
+
+        showPermalink(serverUrl, '', post.id, intl);
+
+        const pressTimeout = setTimeout(() => {
+            pressDetected.current = false;
+            clearTimeout(pressTimeout);
+        }, 300);
+    });
+
     return (
-        <View style={styles.container}>
-            <ChannelInfo post={post}/>
-            <View style={styles.content}>
-                {postAvatar}
-                <View>
-                    {header}
-                    <View style={styles.message}>
-                        <Message
-                            highlight={false}
-                            isEdited={isEdited}
-                            isPendingOrFailed={false}
-                            isReplyPost={false}
-                            location={Screens.MENTIONS}
-                            post={post}
-                            theme={theme}
-                        />
+        <TouchableWithFeedback
+            onPress={handlePress}
+            underlayColor={changeOpacity(theme.centerChannelColor, 0.1)}
+            cancelTouchOnPanning={true}
+        >
+            <View style={styles.container}>
+                <>
+                    <ChannelInfo post={post}/>
+                    <View style={styles.content}>
+                        {postAvatar}
+                        <View style={styles.rightColumn}>
+                            {header}
+                            <View style={styles.message}>
+                                <Message
```

Comment:
I wonder if we need to show file attachments or the other content here.. 

Imagine a post that has a mention and no more text but it has some other type of content like files. if this is not in the design, then probably this needs a UX conversation to be held.